src/components/BulkEnrollmentPage/stepper/AddLearnersStep.test.jsx

```diff
@@ -59,12 +64,25 @@ describe('AddLearnersStep', () => {
     expect(await screen.findByText(mockResults[1].userEmail)).toBeInTheDocument();
     await act(() => mockTableData);
   });
-  it.skip('allows search by email', async () => {
+  it('allows search by email', async () => {
     act(() => {
       renderWithRouter(<StepperWrapper {...defaultProps} />);
     });
     expect(await screen.findByText('Search Email')).toBeInTheDocument();
     await act(() => mockTableData);
+    userEvent.type(screen.getByPlaceholderText('Search Email'), 'beAR');
+    expect(await screen.findByText(/Filtered by userEmail/)).toBeInTheDocument();
+    expect(await screen.findByText('Clear Filters')).toBeInTheDocument();
+    const { subscriptionUUID } = defaultProps;
+    // multiple calls will occur to this function, we only test for the last one
+    // for correctness, and don't test backend filtering part here (tested in backend).
+    await screen.findByDisplayValue('beAR');
+    expect(LicenseManagerApiService.fetchSubscriptionUsers).toHaveBeenLastCalledWith(
```

Comment:
trying to find out how to make this expect pass again, after debounce, it now does not verify that the 'search' was passed

Reply:
asking jest to runAllTimers seems to fix this
also removed useCallback since I already use useMemo now

Reply:
spoke too early, https://github.com/facebook/jest/issues/3465 blocks this right now...

Reply:
yeah so can't find any way to make this work and it looks like there is some known issue with current jest plus lodash.debounce when using real or fake timers. I do not want to spend too much time here and since the test covers that we do call the underlying api call, and mocking out debounce altogether seems to get us past this so going to go with this